src/drive/mobile/lib/reporter.js

```diff
@@ -16,41 +14,36 @@ const getAnalyticsUrl = () => {
 export const ANALYTICS_URL = getAnalyticsUrl()
 
 export const getReporterConfiguration = () => ({
-  shouldSendCallback: () => isEnabled,
+  shouldSendCallback: true,
   environment: __DEVELOPMENT__ ? 'development' : 'production'
 })
 
-export const configureReporter = enable => {
-  isEnabled = enable
+export const configureReporter = () => {
   Raven.config(ANALYTICS_URL, getReporterConfiguration()).install()
 }
 
-export const logException = err => {
+export const logException = (err, extraContext = null) => {
```

Comment:
as `Raven` call it `extra`, shall we stay on this naming `extra` instead of `extraContext`?

Reply:
I called it `extraContext` in the hope it would make its purpose clearer, not sure what's better here